### ISSUE TITLE
Fabricator UI is blocked by joined locker.

### ISSUE THREAD

**Issue by DarkFang123** [Regalis11/Barotrauma#9220]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Opening the fabricator then opening the joint locker will break the UI. You can still craft stuff, but you cant see the name and have to rely on image amd description alone.

### Reproduction steps

1.Open fabricator and close. U can do this as many times as u want. 2. Open the locker and close. 3. Youre screwed because there is no way to undo it.

### Bug prevalence

Happens regularly

### Version

0.17.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by ekqvist** [comment]
Duplicate of #9032. Same steps reported in the comments.
